src/dropsgear.ts

```diff
@@ -0,0 +1,376 @@
+  const fairyMult = fairyMultiplier(meatFamiliar());
+  const fairyBonus = weight * (fairyMult + Math.sqrt(fairyMult) / 2);
+
+  const bjornMeatDropValue = meatValue * (meat + lepBonus);
+  const bjornItemDropValue = itemValue * (item + fairyBonus);
+
+  return bjornMeatDropValue + bjornItemDropValue;
+}
+
+createRiderMode("free", (modifiers: Modifiers) => valueBjornModifiers("free", modifiers), false);
+createRiderMode(
+  "embezzler",
+  (modifiers: Modifiers) => valueBjornModifiers("embezzler", modifiers),
+  true
+);
+createRiderMode("dmt", (modifiers: Modifiers) => valueBjornModifiers("dmt", modifiers), true);
+createRiderMode(
+  "barf",
+  (modifiers: Modifiers) => valueBjornModifiers("barf", modifiers),
+  false,
+  true
+);
+
+/**
+ * Determines the best familiar to bjornify given a particular fight mode
+ * @param mode The BonusEquipMode of this fight: "free", "dmt", "embezzler", or "barf"
+ * @returns The best familiar to bjornify given this fight mode
+ */
+export function pickBjorn(mode: BonusEquipMode = "free"): FamiliarRider {
+  const attempt = pickRider(mode);
+  if (attempt) return attempt;
+  throw new Error("Unable to make a sensible bjorn decision");
+}
+
+const pantsgivingBonuses = new Map<number, number>();
+function pantsgiving() {
+  if (!have($item`Pantsgiving`)) return new Map<Item, number>();
+  const count = get("_pantsgivingCount");
+  const turnArray = [5, 50, 500, 5000];
+  const index =
+    myFullness() === fullnessLimit()
+      ? get("_pantsgivingFullness")
+      : turnArray.findIndex((x) => count < x);
+  const turns = turnArray[index] || 50000;
+
+  if (turns - count > estimatedTurns()) return new Map<Item, number>();
+
+  const cachedBonus = pantsgivingBonuses.get(turns);
+  if (cachedBonus) return new Map([[$item`Pantsgiving`, cachedBonus]]);
+
+  const expectedSinusTurns = getWorkshed() === $item`portable Mayo Clinic` ? 100 : 50;
+  const expectedUseableSinusTurns = globalOptions.ascending
+    ? Math.min(
+        estimatedTurns() - haveEffect($effect`Kicked in the Sinuses`),
+        expectedSinusTurns,
+        estimatedTurns() - (turns - count)
+      )
+    : expectedSinusTurns;
+  const sinusVal = expectedUseableSinusTurns * 1.0 * baseMeat;
+  const fullnessValue =
+    sinusVal +
+    get("valueOfAdventure") * 6.5 -
+    (mallPrice($item`jumping horseradish`) + mallPrice($item`Special Seasoning`));
+  const pantsgivingBonus = fullnessValue / (turns * 0.9);
+  pantsgivingBonuses.set(turns, pantsgivingBonus);
+  return new Map<Item, number>([[$item`Pantsgiving`, pantsgivingBonus]]);
+}
+
+const bestAdventuresFromPants =
+  Item.all()
+    .filter(
+      (item) =>
+        toSlot(item) === $slot`pants` && have(item) && numericModifier(item, "Adventures") > 0
+    )
+    .map((pants) => numericModifier(pants, "Adventures"))
+    .sort((a, b) => b - a)[0] || 0;
+const haveSomeCheese = getFoldGroup($item`stinky cheese diaper`).some((item) => have(item));
```

Comment:
This should probably not be only computed once per garbo run, but I think it is fine to do statically for now

Reply:
We don't take stinky cheese from the stash. unless you mean valuing this, in which case my answer is 'shrug'. We want to cache values because otherwise we get a bunch of tiny microadjustments from turn-to-turn which ruins our equipment caching.